dev/jenkins/Dockerfile-jdk11

```diff
@@ -16,7 +16,7 @@ FROM maven:3.6.3-jdk-11
 # reference: https://github.com/docker-library/openjdk/blob/master/8/jdk/buster/Dockerfile 
 # we need jdk 8 in jdk 11 so that we can compile with jdk 8 and test with jdk 11
 
-ENV JAVA_HOME /usr/local/openjdk-8
+ENV JAVA_HOME /usr/local/openjdk-11
```

Comment:
what is this change for? it should stay as java 8 at this point, see the above comment

Reply:
Attempt to build image locally failed and thought this was the cause but it was unrelated. Reverted this change and am able to build locally fine.